worksheet_A/TerminalHacking.cpp

```diff
@@ -40,7 +54,34 @@ int main()
 	}
 
 	// TODO: implement the rest of the game
```

Comment:
YOU'VE DONE IT SO REMOVE THE COMMENT!!! (Apologies for all caps, this is the 10th time I've typed this today...)